docs/guide/vite.md

```diff
@@ -104,6 +104,209 @@ For debugging production builds, you can enable source maps to see the original
   </template>
 </AposCodeBlock>
 
+### Default Public Build Configuration
+
+While the `apos` build (code in `/ui/apos/` directories) is fully preconfigured, the `public` build (code in `/ui/src/`) comes with minimal configuration to give you more flexibility. Here's what's included by default:
+
+1. **Core Features**:
+   - PostCSS plugin for essential ApostropheCMS features (e.g., "Breakpoint Preview" when enabled)
+   - `Modules/` alias for simplified module imports within `/ui/src/`
+   - `@/` alias for cross-module and cross-build access
+
+2. **Everything Else**:
+   For additional features or frameworks, you'll need to configure them yourself. Common examples include:
+
+   ```javascript
+   // apos.vite.config.js
+   import { defineConfig } from '@apostrophecms/vite/vite';
+   import vue from '@vitejs/plugin-vue';
+   import react from '@vitejs/plugin-react';
+
+   export default defineConfig({
+     plugins: [
+       // Add framework support
+       vue(),
+       // or
+       react(),
+
+       // Add other Vite plugins as needed
+       imagemin({
+         // image optimization options
+       })
+     ],
+     css: {
+       // Additional PostCSS plugins
+       postcss: {
+         plugins: [
+           autoprefixer(),
+           tailwindcss()
+         ]
+       },
+       // Preprocessor options
+       preprocessorOptions: {
+         scss: {
+           additionalData: `$theme: "light";`
+         }
+       }
+     }
+   });
+   ```
+
+### Framework-Specific Configuration
+
+When adding UI frameworks to your project, you'll need to:
+
+1. Install the framework and its Vite plugin:
+   ```bash
+   # For Vue
+   npm install vue @vitejs/plugin-vue
+   
+   # For React
+   npm install react react-dom @vitejs/plugin-react
+   ```
+
+2. Configure the plugin in your Vite config
+3. Follow the framework's best practices for file organization within your `ui/src/` directory
+
+::: tip
+Remember that while you have full control over the public build configuration, it's best to start minimal and add only what you need to keep your build process efficient.
+:::
+
+## Built-in Aliases
```

Comment:
Make a clear recommendation here, new devs are only going to retain one. `Modules/` is required for bc reasons but it sounds like `@/` is just better, especially for DX. So we should relegate `Modules/` to a second section that clearly indicates it's not as good.

Reply:
Reordered and added a sentence.